test/utils/index.js

```diff
@@ -3,17 +3,54 @@
 const delay = require('delay')
 const pRetry = require('p-retry')
 const pTimeout = require('p-timeout')
-const promisify = require('promisify-es6')
+const DuplexPair = require('it-pair/duplex')
```

Comment:
just for consistency
```suggestion
const duplexPair = require('it-pair/duplex')
```